docs/index.md

```diff
@@ -1,11 +1,58 @@
-# What is `earthaccess`?
+<p align="center">
+<img alt="earthaccess, a python library to search, download or stream NASA Earth science data with just a few lines of code" src="https://user-images.githubusercontent.com/717735/205517116-7a5d0f41-7acc-441e-94ba-2e541bfb7fc8.png" width="70%" align="center" />
+</p>
 
-`earthaccess` is a python library to **search for**, and **download** or **stream** NASA Earth science data with just a few lines of code.
 
-Open science only reaches its full potential if we have easy-to-use workflows that facilitate research in an efficient and reproducible way. Unfortunately —as it stands today— scientists and students alike face a steep learning curve adapting to systems that have grown too complex and end up spending more time on the technicalities of the tools, cloud and NASA APIs than focusing on their important science.
+# Welcome to `earthaccess`!
+
+`earthaccess` is a Python library to **search for** and **download** or **stream** NASA Earth science data with just a few lines of code.
+
+>*"earthaccess revolutionizes data access by drastically reducing the complexity and code required. Since open science is a collaborative effort involving people from different technical backgrounds, our team took the approach that data analysis can and should be made more inclusive and accessible by reducing the complexities of underlying systems."*
+>
+>Luis López, an NSIDC software developer and earthaccess creator
+
+# Community Resources
+
+Earthaccess is a highly active, open-source, and community-driven library whose development is being led through NASA Openscapes, an initiative to support researchers using data distributed by NASA Distributed Active Archive Centers (DAACs) as they migrate workflows to the cloud. Development of the earthaccess library also features leadership and contributions from private industry and the broader user community.
```

Comment:
```suggestion
`earthaccess` is a highly active, open-source, and community-driven library developed by [NASA DAACs](https://www.earthdata.nasa.gov/centers), private industry, and the broader NASA data user community.

`earthaccess` development is facilitated by [NASA Openscapes](https://nasa-openscapes.github.io/), an initiative to support researchers using data distributed by NASA Distributed Active Archive Centers (DAACs) as they migrate workflows to the cloud.
```

I felt that the sentence about NASA Openscapes is important, but providing the definition of NASA Openscapes interrupted the idea of "who are the contributors". I felt it deserved its own paragraph! I changed "led" to "facilitated" to be more specific.

Reply:
I feel like "Community resources" (i.e. the bullets below) could have its own page. The paragraphs above could stay in the "About" section in the home page.

Reply:
I like this Matt!